### PR TITLE
More logs for `TestDaemonNoSpaceleftOnDeviceError`

### PR DESCRIPTION
**Pull request by cpuguy83**

### CODE REVIEW

File: integration-cli/docker_cli_daemon_test.go [hunk 1]
```diff
@@ -1825,27 +1825,28 @@ func (s *DockerDaemonSuite) TestDaemonNoSpaceleftOnDeviceError(c *check.C) {
...
-	}
+	out, err := cmd.CombinedOutput()
+	c.Assert(err, checker.IsNil, check.Commentf(string(out)))
+
 	cmd = exec.Command("mkfs.ext4", "-F", "/tmp/testfs.img")
-	if err := cmd.Run(); err != nil {
-		c.Fatalf("mkfs.ext4 failed: %v", err)
-	}
+	out, err = cmd.CombinedOutput()
+	c.Assert(err, checker.IsNil, check.Commentf(string(out)))
```
Comment (thaJeztah) [inline integration-cli/docker_cli_daemon_test.go]: is the "mkfs.ext4 failed:" you removed still useful here? (same as the other ones)

Comment (cpuguy83) [inline integration-cli/docker_cli_daemon_test.go]: Not really. The failure points you to the piece of code that failed.

**Comment by tonistiigi** [comment]
Local result:
```
START: docker_cli_daemon_test.go:1823: DockerDaemonSuite.TestDaemonNoSpaceleftOnDeviceError
START: check_test.go:174: DockerDaemonSuite.SetUpTest
PASS: check_test.go:174: DockerDaemonSuite.SetUpTest	0.001s

docker_cli_daemon_test.go:1833:
    c.Assert(err, checker.IsNil, check.Commentf(string(out)))
... value *exec.ExitError = &exec.ExitError{ProcessState:(*os.ProcessState)(0xc820476020)} ("exit status 1")
... mke2fs 1.42.12 (29-Aug-2014)
/tmp/testfs.img is mounted; will not make a filesystem here!


START: check_test.go:179: DockerDaemonSuite.TearDownTest
PASS: check_test.go:179: DockerDaemonSuite.TearDownTest	0.231s

FAIL: docker_cli_daemon_test.go:1823: DockerDaemonSuite.TestDaemonNoSpaceleftOnDeviceError
```

btw, we don't have a CI that would automatically run it on overaly atm.

**Comment by tonistiigi** [comment]
https://jenkins.dockerproject.org/job/Docker%20Master%20(overlay%20on%20overlay)/2438/consoleFull
```
----------------------------------------------------------------------
FAIL: docker_cli_daemon_test.go:1823: DockerDaemonSuite.TestDaemonNoSpaceleftOnDeviceError

docker_cli_daemon_test.go:1841:
    c.Assert(err, checker.IsNil, check.Commentf(string(out)))
... value *exec.ExitError = &exec.ExitError{ProcessState:(*os.ProcessState)(0xc8207c6b60)} ("exit status 32")
... mount: /tmp/testfs.img: failed to setup loop device: No such file or directory



----------------------------------------------------------------------
```

**Comment by cpuguy83** [comment]
@tonistiigi I get the same issue on docker 4 mac (which is AUFS).

**Comment by cpuguy83** [comment]
btw, totally want this extra logging to be merged

**Comment by estesp** [comment]
I'm fine with the extra test output logging; LGTM

**Comment by vdemeester** [comment]
Same here. LGTM 🐼

**Pull request merged**